### DISCUSSION TITLE
mission control is incompatible with the new version of switch

### DISCUSSION THREAD

**Discussion by brunopepoxd** [ndeadly/MissionControl#672] [General]
The new version of switch 16.0.3 crashes with the current version of mission control 0.9.3 and I would like to know a solution or I will have to wait for the next update compatible with it

**Comment by ndeadly** [comment]
The current release supports all 16.0.x HOS versions. If you're getting a crash after installing the latest version, it could be that you're running too many custom sysmodules or something. I can't really comment further without seeing an error report.

**Reply by brunopepoxd** [reply]
no, I only had that module installed and when you turn on the console this error message appears
![image](https://github.com/ndeadly/MissionControl/assets/41007832/747179f5-0d06-4b11-977c-b5fee09567ad)

**Reply by Masamune3210** [reply]
01000000000000034 is the sysmodule responsible for showing and handling fatal errors. I don't think that mission-control would be, at least directly, causing that to crash by itself.

**Reply by ndeadly** [reply]
Yeah, someone reported a similar issue a while ago. It appeared it was triggered by the presence of mission control somehow, but it's definitely not the root cause. The module doesn't interact with the fatal module at all. IIRC the error was related to fatal encountering an error trying to access the SD filesystem. Why fatal is even invoked I don't know, but the other user was able to get rid of the error by removing their atmosphere folder and reinstalling both atmosphere and mission control.

**Reply by brunopepoxd** [reply]
I already tried deleting and reinstalling the atmosphere folder and it works but when I add mission control to the SD it generates the crash.
I have to try going back to a previous version of atmosphere to see if it is solved

**Reply by Masamune3210** [reply]
What likely causes fatal to be invoked is the same thing that causes fatal to crash. Some module tries to access the sd card, encounters an unknown or malformed state, crashes, fatal gets spawned by the system to handle the crash, and then promptly slips on the same puddle when trying to write the erpt to the sd card. My guess is your sd card is likely either on its way out, your sd card reader is on its way out, or you have some SERIOUS filesystem corruption

**Reply by brunopepoxd** [reply]
Hello, I was able to solve it, thank you very much everyone for helping me .
The conflict occurred in the default folder for downloading mission control since there were previous versions of mission control in it, it did not complete the installation and the files with the previous version were overwritten, which generated a conflict
